pkg/kube/object.go

```diff
@@ -23,6 +23,9 @@ const (
 
 	KindNode Kind = "Node"
 
+	// pseudo Kind
+	KindImage Kind = "Image"
```

Comment:
If you don't prefer defining `KindImage` since it is not a Kubernetes resource, tell me any better way. I'm definitely sure that you are more familiar than me.